diff --git a/dist/card/element/background-card-element.js b/dist/card/element/background-card-element.js
index 37a94805c7c9e68626f54e60ab96f830c964a929..513ae82c4c7b5487852b25c8866e1b77f00389e1 100644
--- a/dist/card/element/background-card-element.js
+++ b/dist/card/element/background-card-element.js
@@ -28,7 +28,8 @@ const OBSERVED_ATTRS = [
     "tilt-degrees",
     "glow-size",
 ];
-export class BackgroundCardElement extends HTMLElement {
+const HTMLElementBase = typeof HTMLElement === "undefined" ? class {} : HTMLElement;
+export class BackgroundCardElement extends HTMLElementBase {
     static get observedAttributes() {
         return OBSERVED_ATTRS;
     }
